routes/utils.js

```diff
@@ -84,11 +84,11 @@ function persistProjectFiles(config, user, project, data, callback) {
           return;
         }
 
-        files[body.path] = {
+        files.push({
           id: body.id,
           project_id: body.project_id,
```

Comment:
So this is what gets sent back to the user! This is not being stored in the session